src/build_infrastructure/android/app.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+
+from aws_cdk import core
+from cdk_build_pipeline.build_pipeline_stack import BuildPipelineStack
+
+app = core.App()
+
+region = app.node.try_get_context("region")
```

Comment:
To debug your problem from the PR description, is this not `None`, here?

Reply:
I added it to the cdk.context.json file and also via the command line parameters. In both instances the variables get set here (expected), but still spit out the error mentioned in the description

Reply:
I created essentially the [same thing in JS](https://gist.github.com/rjuliano/aa19201c542d129bccf937a345817991) and it works, so I'm not sure what I'm missing here.